fakeDataGen.js

```diff
@@ -1,20 +1,30 @@
-const faker = require('faker');
 const mongoose = require('mongoose');
 const fs = require('fs');
 
 // Output array
 const outputArray = [];
 
-// Seed so we can reuse this file and get the same results
-faker.seed(100);
-
 // Create 100 records
 const numRecords = 100;
 for (let i = 1; i <= numRecords; i++) {
-  const numOfImages = faker.random.number({ min: 1, max: 10 });
+  const imageIds = [
+    445,
+    3,
+    225,
+    21,
+    20,
+    201,
+    2,
+    180,
+    160,
+    119
+  ];
+
+  const numOfImages = Math.floor(Math.random() * Math.floor(imageIds.length - 1)) + 1;
```

Comment:
do you need Math.floor for (imageIds.length - 1)? It think those will be integers even before the Math.floor.

Reply:
Nope, I don't, you're totally right. Good catch, thank you!